### PR TITLE
Update dependencies to prepare for release 2.0.0-beta.15

### PR DESCRIPTION
**Pull request by kmadsen**
### Description

Bumping versions

- [x] Bump maps sdk to version 10.0.0-rc.1
- [x] Bump common sdk to version 14.0.1
- [x] Bump nav native version to 53.0.1

It appears we're having more problems with testing these objects. https://github.com/mapbox/mapbox-navigation-android/issues/4492
I ended up testing the native mappers with instrumetation tests here https://github.com/mapbox/mapbox-navigation-android/pull/4488

### CODE REVIEW

File: examples/src/main/java/com/mapbox/navigation/examples/util/RouteDrawingActivity.kt [hunk 1]
```diff
@@ -79,13 +79,12 @@ class RouteDrawingActivity : AppCompatActivity() {
...
         super.onCreate(savedInstanceState)
         setContentView(R.layout.layout_route_drawing_activity)
-        val tileStore = TileStore.getInstance()
+        val tileStore = TileStore.create()
         val mapboxMapOptions = MapInitOptions(this)
         val resourceOptions = ResourceOptions.Builder()
             .accessToken(getMapboxAccessTokenFromResources())
             .assetPath(filesDir.absolutePath)
-            .cachePath(filesDir.absolutePath + "/mbx.db")
-            .cacheSize(100000000L) // 100 MB
```
Comment (kmadsen) [inline examples/src/main/java/com/mapbox/navigation/examples/util/RouteDrawingActivity.kt:88]: wasn't certain about this one. but it's only an example so seems right

**Comment by kmadsen** [comment]
Failing probably because we're early. 
```
Could not determine the dependencies of task ':examples:dataBindingMergeDependencyArtifactsDebug'.
> Could not resolve all task dependencies for configuration ':examples:debugRuntimeClasspath'.
   > Could not find com.mapbox.navigator:mapbox-navigation-native:53.0.1.
     Searched in the following locations:
```

File: examples/src/main/java/com/mapbox/navigation/examples/util/RouteDrawingActivity.kt (hunk 1 above)
Comment (kmadsen) [reply examples/src/main/java/com/mapbox/navigation/examples/util/RouteDrawingActivity.kt:88]: oh it says, so this is good

"Remove ResourceOptions::cacheSize and DefaultAmbientCacheSize constant. (https://github.com/mapbox/mapbox-maps-android/pull/399)"

**Review by Guardiola31337 — APPROVED** [review]
Took this for a quick spin and everything went smoothly 🚀

File: libnavigation-core/src/test/java/com/mapbox/navigation/core/navigator/NavigatorMapperTest.kt [hunk 2]
```diff
@@ -29,8 +29,11 @@ import org.junit.Assert.assertFalse
 import org.junit.Assert.assertNotNull
 import org.junit.Assert.assertNull
 import org.junit.Assert.assertTrue
+import org.junit.Ignore
 import org.junit.Test
 
+// https://github.com/mapbox/mapbox-navigation-android/issues/4492
+@Ignore
```
Comment (kmadsen) [inline libnavigation-core/src/test/java/com/mapbox/navigation/core/navigator/NavigatorMapperTest.kt:36]: cc: @LukasPaczos  @Guardiola31337 

Have a testing with native issue here

**Pull request merged**